requirements-dev.txt

```diff
@@ -15,3 +15,4 @@ pytest-mock==1.10.0
 sphinx-rtd-theme==0.2.4
 crc32c==1.7
 py==1.8.0
+zstandard==0.13.0
```

Comment:
It'd be nice to add a small section to the docs saying that `zstd` is supported with the addition of `zstandard >= 0.13`... I think we've already got something like that for `snappy` that you can use as an example...